spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/spanner/GcpSpannerAutoConfigurationTests.java

```diff
@@ -58,7 +58,6 @@ public void testTestRepositoryCreated() {
 		});
 	}
 
-	@EnableSpannerRepositories
```

Comment:
To answer to your question, @joaoandremartins .
Add here `@AutoConfigurationPackage` instead.

The main point is to let to Spring Boot to know where to find `Repository` (and other similar) classes.
In this case we are lucky that our `TestRepository` in the same package. :smile: